libs/core/src/lib/combobox/combobox.component.ts

```diff
@@ -128,7 +125,7 @@ export class ComboboxComponent implements ControlValueAccessor, OnInit, OnChange
 
     /** Max height of the popover. Any overflowing elements will be accessible through scrolling. */
     @Input()
-    maxHeight: string = '200px';
+    maxHeight: string = '50vh';
 
```

Comment:
Could you add also `inputTextChange` output? It would throw `inputText` everytime it's changed. Sometimes developers refresh api calls on input change, which is not currently possible on `objectMode`.

Reply:
Wouldn't they just use ngModelChange?

Reply:
Nope, on `[objectMode]` ngModelChange is not triggered on text input change, it's triggered only, when the option is choosen.